langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaOpenAiStreamingChatModelIT.java

```diff
@@ -47,5 +52,73 @@ void should_generate_response() {
         assertThat(response.finishReason()).isEqualTo(STOP);
     }
 
+    @Test
+    void should_listen_request_and_response() {
```

Comment:
I am not sure this test is required, it is already tested in `OpenAiStreamingChatModelIT`

Reply:
hmmm. Maybe it's no need to test it.